scripts/updateSources.js

```diff
@@ -159,27 +123,15 @@ async function main() {
       return content;
     }
 
-    // Dialog: Workaround for https://github.com/vaadin/vaadin-confirm-dialog-flow/issues/136
-    // Since this project contains a dependency to vaadin-confirm-dialog, the height is different
-    // and the tests fail.
-    content = ignore_test_method(content, source, 'DialogIT.openAndCloseBasicDialog_labelRendered');
     content = ignore_test_method(content, source, 'ServerSideEventsIT.chartClick_occured_eventIsFired');
-    content = ignore_test_method(content, source, 'GridDetailsRowIT.gridUpdateItemUpdateDetails');
-    content = ignore_test_method(content, source, 'BasicIT.customComboBox_circularReferencesInData_isEdited');
-    content = ignore_test_method(content, source, 'BasicIT.customComboBoxIsUsedForEditColumn');
-    content = ignore_test_method(content, source, 'BasicIT.checkboxEditorIsUsedForCheckboxColumn');
```

Comment:
These ITs have been re-enabled in #280, so there is no more need to ignore them here.